packages/fxa-auth-server/docs/swagger/account-docs.ts

```diff
@@ -0,0 +1,247 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Just my opinion, I think you could probably take off `-docs` on these files since it's already inside of a directory called `docs` 🤔 or use `-api` instead.